### PR TITLE
add small glossary into readme

### PR DESCRIPTION
**Pull request by Friz-zy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,12 @@ If you wish to proceed without Vagrant (not recommended!), go straight to
 
 ## Contents
 
+[Terminology](https://docs.ansible.com/ansible/glossary.html):
+ - [command or action](https://docs.ansible.com/ansible/intro_adhoc.html): [ansible module](https://docs.ansible.com/ansible/modules.html) or just a shell command. Intro in [step-02](https://github.com/leucos/ansible-tuto/tree/master/step-02).
+ - task: it's combine an action (a module and its arguments) with a name and optionally some other keywords (like looping directives).
+ - [palybook](https://docs.ansible.com/ansible/playbooks_intro.html): yaml file that describes the algorithm of execution tasks. Intro in [step-04](https://github.com/leucos/ansible-tuto/tree/master/step-04).
+ - [role](https://docs.ansible.com/ansible/playbooks_roles.html): directory that contain all files, templates and playbooks; units of organization like python module. Intro in [step-12](https://github.com/leucos/ansible-tuto/tree/master/step-12).
```
Comment (leucos) [inline README.md]: Well, a role is more an organisational unit grouping tasks together in order to install a piece of software. And a role does not contain a playbook, it is the opposite

File: README.md [hunk 2]
```diff
@@ -84,6 +84,12 @@ If you wish to proceed without Vagrant (not recommended!), go straight to
 
 ## Contents
 
+[Terminology](https://docs.ansible.com/ansible/glossary.html):
+ - [command or action](https://docs.ansible.com/ansible/intro_adhoc.html): [ansible module](https://docs.ansible.com/ansible/modules.html) or just a shell command. Intro in [step-02](https://github.com/leucos/ansible-tuto/tree/master/step-02).
+ - task: it's combine an action (a module and its arguments) with a name and optionally some other keywords (like looping directives).
+ - [palybook](https://docs.ansible.com/ansible/playbooks_intro.html): yaml file that describes the algorithm of execution tasks. Intro in [step-04](https://github.com/leucos/ansible-tuto/tree/master/step-04).
```
Comment (leucos) [inline README.md]: palybook -> playbook
I'd say: A playbook contains roles executed in sequence, and eventually individual tasks

File: README.md [hunk 3]
```diff
@@ -84,6 +84,12 @@ If you wish to proceed without Vagrant (not recommended!), go straight to
 
 ## Contents
 
+[Terminology](https://docs.ansible.com/ansible/glossary.html):
+ - [command or action](https://docs.ansible.com/ansible/intro_adhoc.html): [ansible module](https://docs.ansible.com/ansible/modules.html) or just a shell command. Intro in [step-02](https://github.com/leucos/ansible-tuto/tree/master/step-02).
```
Comment (leucos) [inline README.md]: I'd rather talk about tasks, explaining that a task is the execution of a module, so we don't confuse readers. There is no 'shell command' concept either. Everything goes thru modules (eventually, the shell module, the command module, the X module).

**Comment by leucos** [comment]
@Friz-zy thanks for the PR. You might want to peek at the inline notes above. I can correct these myself if you're not willing to. Thanks for the contribution !

**Comment by Friz-zy** [comment]
Thanks for your work! ;)
I'll fix it myself soon.
Also I plan write two steps: about vagrant provision and ansible galaxy.
Should I add issues about it and put it after existing TBD steps?

**Comment by leucos** [comment]
Sure, go ahead.

**Comment by leucos** [comment]
Oh, sorry for overlooking this !

**Comment by leucos** [comment]
Thanks @Friz-zy !

**Comment by Friz-zy** [comment]
Don't worry, I fully understand you ;)
Thanks for merging

**Pull request merged**